gaphor/SysML/toolbox.py

```diff
@@ -31,6 +25,31 @@ def metaclass_config(new_item):
     new_item.subject.name = "Class"
 
 
+def create_association(
+    assoc_item: uml_items.AssociationItem, part_association: bool
```

Comment:
I would mot use a boolean here. Just the type of association as a str, or maybe Enum.

Reply:
Thanks added an Enum :+1: